UPGRADE-3.x.md

```diff
@@ -40,8 +40,14 @@ You MUST use Symfony's [`help`](https://symfony.com/doc/4.4/reference/forms/type
 Before:
 ```php
 $formMapper
-    ->add('field', null, [
+    ->add('field', null, [], [
         'help' => 'Help text <small>Please!</small>',
+    ])
+    ->add('field2')
+    ->addHelp('field2', 'This field is required.')
```

Comment:
isnt this deprecated? IMo we should not add docs about deprecated things. What is the correct way to add helps now?

Reply:
It is before - after example